### PR TITLE
GCP IoT Core Compatibility docs

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/modules/GCP_device.md [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+Received message b'\x00\x01\x02\x03\x04\x05\x06\x07\x08\t\n\x0b\x0c\r\x0e\x0f\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~\x7f\x80\x81\x82\x83\x84\x85\x86\x87\x88\x89\x8a\x8b\x8c\x8d\x8e\x8f\x90\x91\x92\x93\x94\x95\x96\x97\x98\x99\x9a\x9b\x9c\x9d\x9e\x9f\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf\xb0\xb1\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xbb\xbc\xbd\xbe\xbf\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf\xd0\xd1\xd2\xd3\xd4\xd5\xd6\xd7\xd8\xd9\xda\xdb\xdc\xdd\xde\xdf\xe0\xe1\xe2\xe3\xe4\xe5\xe6\xe7\xe8\xe9\xea\xeb\xec\xed\xee\xef\xf0\xf1\xf2\xf3\xf4\xf5\xf6\xf7\xf8\xf9\xfa\xfb\xfc\xfd\xfe\xff' on topic /devices/c2-ec-x509/config with qos 1
+```
+
+This is just the same as before, but now you are using EMQX instead of Google Cloud IoT Core.
+
+## Additional APIs
+
+EMQX provides some additional API calls to manage EMQX data using "device" terminology.
+
+### Manage Individual Device Configuration
```
Comment (savonarola) [inline en_US/modules/GCP_device.md]: Is there a Dashboard interface for device management? Should we document it?

**Review by savonarola — DISMISSED** [review]

File: en_US/modules/GCP_device.md (hunk 1 above)
Comment (Meggielqk) [reply en_US/modules/GCP_device.md]: Yes, there is a module called "GCP IoT Core Device" on Dashboard. I have added the Dashboard descriptions in the section "Migrate and Manage Devices via Dashboard" in the document.

File: zh_CN/modules/GCP_device.md [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+
+1. 在**模块**页面上，点击 **GCP IoT Core 设备**的**管理**按钮。
+
+   <img src="./assets/GCP_device_manage.png" alt="GCP_device_manage" style="zoom:67%;" />
+
+2. 在**详情**页面的**设备管理**页签，点击**导入**以批量导入设备数据或者点击**添加**以手动添加设备数据。
+
+   - 如果点击**导入**，将弹出对话框让您导入从 GCP IoT Core 中导出的 json 文件。选中文件并点击**打开**。
+
+     <img src="/Users/emqx/Documents/GitHub/emqx-docs/en_US/modules/assets/GCP_device_import.png" alt="GCP_device_import" style="zoom:67%;" />
```
Comment (wivwiv) [inline zh_CN/modules/GCP_device.md]: bad image path

File: zh_CN/modules/GCP_device.md [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+Password is eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE2ODIzNDE2NzgsImV4cCI6MTY4MjM0Mjg3OCwiYXVkIjoiaW90LWV4cG9ydCJ9.04_zR71fmi0YikSxZbb_wxpVTnikt2XIkxkuI6JM6VS0VJ1B8QrggHuUron8MAOSJDJu9SVa2fuuFFjJEKJ-Bw
+Subscribing to config topic /devices/c2-ec-x509/config
+on_connect Connection Accepted.
+Received message b'\x00\x01\x02\x03\x04\x05\x06\x07\x08\t\n\x0b\x0c\r\x0e\x0f\x10\x11\x12\x13\x14\x15\x16\x17\x18\x19\x1a\x1b\x1c\x1d\x1e\x1f !"#$%&\'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~\x7f\x80\x81\x82\x83\x84\x85\x86\x87\x88\x89\x8a\x8b\x8c\x8d\x8e\x8f\x90\x91\x92\x93\x94\x95\x96\x97\x98\x99\x9a\x9b\x9c\x9d\x9e\x9f\xa0\xa1\xa2\xa3\xa4\xa5\xa6\xa7\xa8\xa9\xaa\xab\xac\xad\xae\xaf\xb0\xb1\xb2\xb3\xb4\xb5\xb6\xb7\xb8\xb9\xba\xbb\xbc\xbd\xbe\xbf\xc0\xc1\xc2\xc3\xc4\xc5\xc6\xc7\xc8\xc9\xca\xcb\xcc\xcd\xce\xcf\xd0\xd1\xd2\xd3\xd4\xd5\xd6\xd7\xd8\xd9\xda\xdb\xdc\xdd\xde\xdf\xe0\xe1\xe2\xe3\xe4\xe5\xe6\xe7\xe8\xe9\xea\xeb\xec\xed\xee\xef\xf0\xf1\xf2\xf3\xf4\xf5\xf6\xf7\xf8\xf9\xfa\xfb\xfc\xfd\xfe\xff' on topic /devices/c2-ec-x509/config with qos 1
+```
+
+结果和之前相同，但是现在您使用的是 EMQX 而不是 Google Cloud IoT Core。
+
+## 限制
```
Comment (wivwiv) [inline zh_CN/modules/GCP_device.md:473]: I think it's needs to be removed or change to a more friendly description. This is a negative description may influence users to choose EMQX.

File: zh_CN/modules/modules.md [hunk 4]
```diff
@@ -44,6 +44,7 @@ EMQX 发行包中提供了丰富了功能模块，包括 认证鉴权、协议
   - [MQTT 主题重写](./topic_rewrite.md)
   - [MQTT 保留消息](./retainer.md)
   - [MQTT 延迟发布](./delayed_publish.md)
+  - [GCP IoT Core 设备](./GCP_device.md)
```
Comment (wivwiv) [inline zh_CN/modules/modules.md]: Adjustment order Refer to directory_ee.json

**Review by wivwiv — CHANGES_REQUESTED** [review]

**Review by savonarola — DISMISSED** [review]

File: zh_CN/modules/GCP_device.md (hunk 2 above)
Comment (Meggielqk) [reply zh_CN/modules/GCP_device.md]: fixed

File: zh_CN/modules/GCP_device.md (hunk 3 above)
Comment (Meggielqk) [reply zh_CN/modules/GCP_device.md:473]: Updated the description

**Review by wivwiv — APPROVED** [review]

**Pull request merged**